### DISCUSSION TITLE
Suggestion: New image name

### DISCUSSION THREAD

**Discussion by ksurl** [haugene/docker-transmission-openvpn#2038] [Ideas]
Just a thought, when wireguard hits stable release for this image, maybe add an alternate tag name along with haugene/transmission-openvpn like haugene/transmission-vpn. and then maybe drop openvpn tag at the next major release. I'm guessing wireguard will go stable at 4.x so 5.0 seems like a reasonable point to drop the openvpn tag with a release note disclaimer

**Comment by haugene** [comment]
Time has passed and I've made a new attempt at the Wireguard support. See this discussion thread which also handles the image and repo naming: https://github.com/haugene/docker-transmission-openvpn/discussions/2608
